### PR TITLE
Release 2020 Dec 14

### PR DESCRIPTION
**Pull request by aminlatifi**

### CODE REVIEW

File: src/services/csv/source.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = app => {
           'migratedProjectId',
           'createdAt',
           'ownerAddress',
-          'token',
+          'tokenSymbol',
```
Comment (aminlatifi) [inline src/services/csv/source.js:23]: @mohammadranjbarz  CSV will not work after this

Comment (mohammadranjbarz) [reply src/services/csv/source.js:23]: > @mohammadranjbarz CSV will not work after this

I'll check it.

**Comment by mohammadranjbarz** [comment]
@aminlatifi 
after merging this to master we should run migration scripts to fill donations and milestones tokenAddress ,
Don't forget it

**Comment by GriffGreen** [comment]
not going to merge until develop is working....

**Pull request closed without merging**